lib/v2/thepaper/utils.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
                 title: contentDetail.name || contentDetail.shareName,
                 link: itemUrl,
                 description,
-                category: [...(contentDetail.tagList?.map((t) => t.tag) ?? []), ...(contentDetail?.nodeInfo?.name ?? [])],
+                category: [].concat(contentDetail.tagList?.map((t) => t.tag) ?? [], contentDetail?.nodeInfo?.name ?? []),
```

Comment:
https://github.com/DIYgod/RSSHub/pull/14383/files/2a3274c0bd1a5f802f6edcdc2ad410b6f8952db7
```suggestion
                category: [...(contentDetail.tagList?.map((t) => t.tag) ?? []), contentDetail?.nodeInfo?.name ?? []],
```